### ISSUE TITLE
Trigger word is not adding causing it not to start unless manually added

### ISSUE THREAD

**Issue by Ghost4242** [cocktailpeanut/fluxgym#77]
So for some reason when you type your trigger word in the trigger word box it doesn't add it to the captions of images which then when you press training make it crash with some vague error but if you manually add the trigger word under every image in the gui it then starts

**Comment by FruitPigFoot** [comment]
yeah me too. most of the times, it won't add trigger words automatically
